core/includes/image.inc

```diff
@@ -584,18 +584,21 @@ function image_is_svg($uri) {
 function image_add_svg_attributes($svg_content, array $attributes) {
   $doc = new DOMDocument();
   $doc->loadXML($svg_content);
+  $svg_tag = $doc->getElementsByTagName('svg')->item(0);
 
   // Convert the alt attribute to a <title> element.
   if (isset($attributes['alt'])) {
       try {
         if (strlen($attributes['alt'])) {
           $title = $doc->createElement('title');
           $title->textContent = $attributes['alt'];
-          $doc->firstChild->prepend($title);
+          // Since DOMDocument::prepend() is not available in PHP versions prior
+          // to v8, we are using DOMNode::insertBefore().
+          $svg_tag->insertBefore($title, $svg_tag->firstChild);
         }
         // Remove any given <title> element if alt is an empty string.
-        elseif ($doc->firstChild->firstChild && $doc->firstChild->firstChild->nodeName === 'title') {
-          $doc->firstChild->removeChild($doc->firstChild->firstChild);
+        elseif ($svg_tag->firstChild && $svg_tag->firstChild->nodeName === 'title') {
+          $svg_tag->removeChild($svg_tag->firstChild);
```

Comment:
I've tested this by applying it to the PR branch for https://github.com/backdrop/backdrop/pull/4769 and PHPCS complained about wrong indentation:
<img width="1661" alt="image" src="https://github.com/backdrop/backdrop/assets/2423362/da583c51-aefa-48f3-9b47-395fd3b75717">

I believe that it is the inline comment before the `elseif` that is tripping it up. Can we please move it inside the `elseif`, above this line?

Reply:
...actually, there's more to it: the entire `try` within the `if (isset($attributes['alt'])) {` is indented wrong. This is what fixed the PHPCS nag in my PR: https://github.com/backdrop/backdrop/pull/4769/commits/54d0f97b0d99a0b212a2dcf90917cb1b0c112a6b

Reply:
I see. But that's not "my" code. Didn't touch it in any way. That's why our phpcs action doesn't nag.

Touching it now would introduce an unrelated change. Do you insist in an unrelated change here? It'll make it harder to figure out, what the intention of that code was by using git blame. Which is the main reason, why I strive to avoid unrelated changes in the first place.